File: LABORATORIOS/LABORATORIO1/Ejercicio2/Java/CalcularPromedio.java
import java.util.Scanner;

public class CalcularPromedio {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Ingrese la primera nota (0-100):");
        int nota1 = scanner.nextInt();

        System.out.println("Ingrese la segunda nota (0-100):");
        int nota2 = scanner.nextInt();

        System.out.println("Ingrese la tercera nota (0-100):");
        int nota3 = scanner.nextInt();

        double promedio = (nota1 + nota2 + nota3) / 3.0;

        if (promedio >= 0 && promedio <= 50) {
            System.out.println("Reprobado");
        } else if (promedio >= 51 && promedio <= 69) {
            System.out.println("Aprobado y regular");
        } else if (promedio >= 70 && promedio <= 79) {
            System.out.println("Aprobado y bueno");
        } else if (promedio >= 80 && promedio <= 89) {
            System.out.println("Aprobado y muy bueno");
        } else if (promedio >= 90 && promedio <= 100) {
            System.out.println("Aprobado y excelente");
        } else {
            System.out.println("Promedio inválido");
        }
    }
}

File: LABORATORIOS/LABORATORIO1/Ejercicio2/Python/CalcularPromedio.py
print("Ingrese la primera nota (0-100):")
nota1 = int(input())

print("Ingrese la segunda nota (0-100):")
nota2 = int(input())

print("Ingrese la tercera nota (0-100):")
nota3 = int(input())

promedio = (nota1 + nota2 + nota3) / 3.0

if 0 <= promedio <= 50:
    print("Reprobado")
elif 51 <= promedio <= 69:
    print("Aprobado y regular")
elif 70 <= promedio <= 79:
    print("Aprobado y bueno")
elif 80 <= promedio <= 89:
    print("Aprobado y muy bueno")
elif 90 <= promedio <= 100:
    print("Aprobado y excelente")
else:
    print("Promedio inválido")